src/Stormpath/Http/HttpClientRequestExecutor.php

```diff
@@ -29,10 +30,10 @@ class HttpClientRequestExecutor implements RequestExecutor
     private $httpClient;
     private $signer;
 
-    public function __construct()
+    public function __construct(RequestSigner $signer)
     {
         $this->httpClient = new Client();
```

Comment:
PHP will throw an exception if null is passed or it is any class that does not implement Request Signer